cmd/erasure-server-pool-rebalance.go

```diff
@@ -810,6 +810,9 @@ func (z *erasureServerPools) StartRebalance() {
 		}
 
 		go func(idx int) {
+			for newObjectLayerFn() == nil {
```

Comment:
@jiuker currently it seems that the code that calls StartBalance() will always have the object layer initialized. However, if you want to add more defensive code, you can do that in the beginning of this function